defaults/main.yml

```diff
@@ -24,7 +24,8 @@ haproxy_rsyslog_conf_path: /etc/rsyslog.d/49-haproxy.conf
 haproxy_logrotate_managed: true
 haproxy_logrotate_conf_template: logrotate_haproxy.conf.j2
 haproxy_logrotate_template_conf_path: /etc/logrotate.d/haproxy
-haproxy_logrotate_retention: 5
+haproxy_logrotate_retention: "{{ openio_log_retention | default(14) | int }}"
```

Comment:
Is this already used somewhere ? at least it is not in this PR...

Reply:
https://github.com/open-io/ansible-role-openio-haproxy/pull/38/files#diff-eef2403d85493da2c1e1ce382f3669ceR3

Reply:
OK